data/json/items/comestibles/junkfood.json

```
@@ -1135,6 +1135,20 @@
     "fun": 12,
     "vitamins": [ [ "vitC", 85 ], [ "calcium", 22 ], [ "iron", 15 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "hotdogs_newyork",
+    "name": { "str": "new york wiener" },
+    "copy-from": "hotdogs_cooked",
+    "weight": "568 g",
+    "looks-like": "hotdogs_cooked",
```

It's `looks_like`, underscore.
```suggestion
    "looks_like": "hotdogs_cooked",
```